### PR TITLE
PR: Feature/birthdate UI

### LINKED ISSUE #3: Enter your date of birth screen UI
Create UI screen.
Then add the form, which contains date input and submit button.

### PR DESCRIPTION
**Pull request by RobinKim-SWEngineer**
## What is it about?
 - Adding birth date input form with calculation button

## How is it written?
 It consists of two major Expanded widgets, one containing an image and sentences and another containing datePicker and calculation button.

## How to test?
 - Click datePicker, and the result will be reflected in the textButton

Birthdate Input UI

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -1,113 +1,11 @@
 import 'package:flutter/material.dart';
+import 'package:demo/Screens/BirthInput/birthdateInput.dart';
```
Comment (AndreyKalashnik) [inline lib/main.dart]: Let's make a naming in a lowercase, like this : packages:demo/screens/birthInput/bithdate_input_screen.dart

File: lib/Screens/BirthInput/Body.dart [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+    return SafeArea(
+      child: SizedBox(
+        width: double.infinity,
+        child: Column(
+          children: [
+            SizedBox(
+              height: 40,
+            ),
+            Expanded(
+              flex: 3,
```
Comment (AndreyKalashnik) [inline lib/Screens/BirthInput/Body.dart]: Did you put `flex: 3 ` and `flex: 2` in order to make a ration 3:2 ?

File: lib/Screens/BirthInput/Body.dart [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+                            'Calculate',
+                            style: TextStyle(
+                              fontSize: 20,
+                            ),
+                          ),
+                          style: TextButton.styleFrom(
+                              primary: Colors.white,
+                              backgroundColor: Colors.deepOrange,
+                              shape: RoundedRectangleBorder(
+                                  borderRadius:
```
Comment (AndreyKalashnik) [inline lib/Screens/BirthInput/Body.dart]: put some love with commas :) like BorderRadius.all(Radius.circular(15)),),),

File: lib/Screens/BirthInput/Body.dart [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+      setState(() {
+        _selectedDate = pickedDate;
+        _showSnackBarMessage(
+            message: 'Your birth date is selected!', context: context);
+      });
+    }
+  }
+
+  ScaffoldFeatureController _showSnackBarMessage(
+      {required String message, required BuildContext context}) {
```
Comment (AndreyKalashnik) [inline lib/Screens/BirthInput/Body.dart]: if you put a comma after context it would be more readable ;)

File: test/widget_test.dart [hunk 5]
```diff
@@ -13,7 +14,7 @@ import 'package:demo/main.dart';
 void main() {
   testWidgets('Counter increments smoke test', (WidgetTester tester) async {
     // Build our app and trigger a frame.
-    await tester.pumpWidget(MyApp());
```
Comment (AndreyKalashnik) [inline test/widget_test.dart:16]: probably this test will fail :)
you can remove it for now.

**Review by AndreyKalashnik — CHANGES_REQUESTED** [review]
There are small comments to resolve, but overall looks fantastic!! You nailed it fast and accurate!

File: lib/main.dart (hunk 1 above)
Comment (RobinKim-SWEngineer) [reply lib/main.dart]: I see! When naming libraries, I need to use **lowercase_with_underscore** :)

File: lib/Screens/BirthInput/Body.dart (hunk 2 above)
Comment (RobinKim-SWEngineer) [reply lib/Screens/BirthInput/Body.dart]: Yes, the idea is to make the portion 3:2. I saw this from the [youtube link](https://www.youtube.com/watch?v=YEJPg2jwzI8) that you sent.

File: test/widget_test.dart (hunk 5 above)
Comment (RobinKim-SWEngineer) [reply test/widget_test.dart:16]: commented out :)

File: lib/Screens/BirthInput/Body.dart (hunk 2 above)
Comment (AndreyKalashnik) [reply lib/Screens/BirthInput/Body.dart]: Cool!

File: lib/main.dart (hunk 1 above)
Comment (AndreyKalashnik) [reply lib/main.dart]: right! :)

**Review by AndreyKalashnik — APPROVED** [review]
Wonderful !

**Pull request merged**